apps/files_sharing/src/components/SharingEntryLink.vue

```diff
@@ -483,6 +507,7 @@ export default {
 		 * Create a new share link and append it to the list
 		 */
 		async onNewLinkShare() {
+			console.debug('onNewLinkShare called (this.share status)', this.share)
```

Comment:
Please use the logger. The logger respects the configured logging level. 

```suggestion
			this.logger.debug('onNewLinkShare called (this.share status)', { share: this.share })
```